news/2023/2023-05-29-osu-philippines-conquest-festival-2023.md

```diff
@@ -4,26 +4,26 @@ title: "osu! Philippines: CONQuest Festival 2023"
 date: 2023-05-29 07:00:00 +0000
 ---
 
-The osu! Philippines Community is excited to announce its second (unofficial) osu! community-ran space at [CONQuest Festival 2023](https://conquestph.com), which will be held at the SMX Convention Centre in Pasay City, Philippines!
+The osu! Philippines Community is excited to announce its second (unofficial) community-run osu! space at [CONQuest Festival 2023](https://conquestph.com), which will be held at the SMX Convention Centre in Pasay City, Philippines!
 
-![Banner](/wiki/shared/news/2023-05-29-osu-philippines-conquest-festival-2023/banner.png)
+![](/wiki/Tournaments/oPQS/2/img/banner.png)
 
-The osu! booth will have games, goodies, and more as the Filipino community; players and non-players alike, get to know and enjoy osu! more!
+The osu! booth will have games, goodies, and more as the Filipino community, players and non-players alike, get to know and enjoy osu! more!
 
 ## Tournament
 
-The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*-styled Brawl format. The tournament will take place on Day 1 and Day 2, with prizes in store!
+The main event of the space is the **osu! Philippines Quickfire Showdown 2023**. A lineup of confirmed players will go through a LAN tournament featuring a *Super Smash Bros.*–tyled brawl format. The tournament will take place on day 1 and day 2, with prizes in store!
 
 For more information about the tournament, refer to [this wiki article](/wiki/Tournaments/oPQS/2).
 
-## Booth Activties
+## Booth activties
 
-The tournament isn't the only thing that's in store for the attendees! If the burning flames of the competition is not their thing, other activities can be done: for community members that want to meet up with their friends or for those meeting for the first time, this is their chance to create new memories and friends at the booth's Player Hub! Newcomers and those already familiar with the game will be able to also play osu!(lazer) at the booth as well.
+The tournament isn't the only thing that's in store for the attendees! If the burning flames of the competition is not to their thing, there are other activities as well: for community members that want to meet up with their friends or for those meeting for the first time, this is their chance to create new memories and friends at the booth's Player Hub! Newcomers and those already familiar with the game will also be able to play osu!(lazer) at the booth as well.
```

Comment:
> If the burning flames of the competition is not to their thing

original sounds more natural. "not your/their thing" is a pretty common casual phrase

Reply:
oh i somehow misread "thing" as "liking". you're entirely correct

```suggestion
The tournament isn't the only thing that's in store for the attendees! If the burning flames of the competition is not their thing, there are other activities as well: for community members that want to meet up with their friends or for those meeting for the first time, this is their chance to create new memories and friends at the booth's Player Hub! Newcomers and those already familiar with the game will also be able to play osu!(lazer) at the booth as well.
```